### ISSUE TITLE
[GLPI 10] Broken dashboard

### ISSUE THREAD

**Issue by AKorezin** [glpi-project/glpi#11878]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.1

### Bug description

_All_ dashboard displayed incorrectly if selected after _Dashboard_ dashboard.
This issue appeared in `10.0.0` and was not fixed in `10.0.1`.
![How broken dashboard looks](https://user-images.githubusercontent.com/5399964/173131445-2126104b-f649-4169-b44f-eed689dbb139.png)

### Relevant log output

[Log from browser console](https://github.com/glpi-project/glpi/files/8881276/localhost-1654885738624.log)
[Apache log](https://github.com/glpi-project/glpi/files/8881275/apache.log)

### Page URL

`/front/central.php`

### Steps To reproduce

1. Install clean `glpi`;
1. login as `glpi:glpi` you will be at `/front/central.php` page;
1. open _all_ dashboard;
1. observe broken dashboard.

### Your GLPI setup information

<details>
<summary>Information about system installation and configuration</summary>

```
GLPI 10.0.1 ( => /var/www/html)
Installation mode: TARBALL
Current language:en_GB
```

</details>

<details>
<summary>Server</summary>

```
Operating system: Linux a95666ad7c70 5.18.2-arch1-1 #1 SMP PREEMPT_DYNAMIC Mon, 06 Jun 2022 19:58:58 +0000 x86_64
PHP 8.1.7 apache2handler (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apache2handler, bz2, ctype, curl, date,
	dom, exif, fileinfo, filter, ftp, gd, hash, iconv, igbinary, intl, json, ldap, libxml, mbstring, memcached, mysqli, mysqlnd,
	openssl, pcntl, pcre, pdo_sqlite, posix, session, sodium, sqlite3, standard, tokenizer, xml, xmlreader, xmlwriter, zip, zlib)
Setup: max_execution_time="600" memory_limit="64M" post_max_size="8M" safe_mode="" session.save_handler="memcached"
	upload_max_filesize="2M" 
Software: Apache ()
	Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.5005.115 Safari/537.36
Server Software: mariadb.org binary distribution
	Server Version: 10.7.4-MariaDB-1:10.7.4+maria~focal
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: glpi@mariadb/glpi
	Host info: mariadb via TCP/IP
	
PHP version (8.1.7) is supported.
Sessions configuration is OK.
Allocated memory is sufficient.
mysqli extension is installed.
Following extensions are installed: dom, fileinfo, json, simplexml.
curl extension is installed.
gd extension is installed.
intl extension is installed.
libxml extension is installed.
zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (10.7.4) is supported.
The log file has been created successfully.
Write access to /var/www/html/files/_cache has been validated.
Write access to /var/www/html/config has been validated.
Write access to /var/www/html/files/_cron has been validated.
Write access to /var/www/html/files has been validated.
Write access to /var/www/html/files/_dumps has been validated.
Write access to /var/www/html/files/_graphs has been validated.
Write access to /var/www/html/files/_lock has been validated.
Write access to /var/www/html/files/_pictures has been validated.
Write access to /var/www/html/files/_plugins has been validated.
Write access to /var/www/html/files/_rss has been validated.
Write access to /var/www/html/files/_sessions has been validated.
Write access to /var/www/html/files/_tmp has been validated.
Write access to /var/www/html/files/_uploads has been validated.
Web access to files directory is protected
exif extension is installed.
ldap extension is installed.
openssl extension is installed.
zip extension is installed.
bz2 extension is installed.
Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/html/marketplace has been validated.
Timezones seems loaded in database.
```

</details>

<details>
<summary>GLPI constants</summary>

```
GLPI_ROOT: /var/www/html
GLPI_CONFIG_DIR: /var/www/html/config
GLPI_VAR_DIR: /var/www/html/files
GLPI_MARKETPLACE_DIR: /var/www/html/marketplace
GLPI_USE_CSRF_CHECK: 1
GLPI_CSRF_EXPIRES: 7200
GLPI_CSRF_MAX_TOKENS: 100
GLPI_USE_IDOR_CHECK: 1
GLPI_IDOR_EXPIRES: 7200
GLPI_ALLOW_IFRAME_IN_RICH_TEXT: 
GLPI_TELEMETRY_URI: https://telemetry.glpi-project.org
GLPI_INSTALL_MODE: TARBALL
GLPI_NETWORK_MAIL: glpi@teclib.com
GLPI_NETWORK_SERVICES: https://services.glpi-network.com
GLPI_MARKETPLACE_ALLOW_OVERRIDE: 1
GLPI_MARKETPLACE_MANUAL_DOWNLOADS: 1
GLPI_USER_AGENT_EXTRA_COMMENTS: 
GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE: 1
GLPI_AJAX_DASHBOARD: 1
GLPI_CALDAV_IMPORT_STATE: 0
GLPI_DEMO_MODE: 0
GLPI_CENTRAL_WARNINGS: 1
GLPI_DOC_DIR: /var/www/html/files
GLPI_CACHE_DIR: /var/www/html/files/_cache
GLPI_CRON_DIR: /var/www/html/files/_cron
GLPI_DUMP_DIR: /var/www/html/files/_dumps
GLPI_GRAPH_DIR: /var/www/html/files/_graphs
GLPI_LOCAL_I18N_DIR: /var/www/html/files/_locales
GLPI_LOCK_DIR: /var/www/html/files/_lock
GLPI_LOG_DIR: /var/www/html/files/_log
GLPI_PICTURE_DIR: /var/www/html/files/_pictures
GLPI_PLUGIN_DOC_DIR: /var/www/html/files/_plugins
GLPI_RSS_DIR: /var/www/html/files/_rss
GLPI_SESSION_DIR: /var/www/html/files/_sessions
GLPI_TMP_DIR: /var/www/html/files/_tmp
GLPI_UPLOAD_DIR: /var/www/html/files/_uploads
GLPI_INVENTORY_DIR: /var/www/html/files/_inventories
GLPI_NETWORK_REGISTRATION_API_URL: https://services.glpi-network.com/api/registration/
GLPI_MARKETPLACE_PLUGINS_API_URI: https://services.glpi-network.com/api/glpi-plugins/
GLPI_I18N_DIR: /var/www/html/locales
GLPI_VERSION: 10.0.1
GLPI_SCHEMA_VERSION: 10.0.1
GLPI_MARKETPLACE_PRERELEASES: 
GLPI_MIN_PHP: 7.4.0
GLPI_MAX_PHP: 8.2.0
GLPI_YEAR: 2022
```

</details>

<details>
<summary>Libraries</summary>

```
htmlawed/htmlawed version 1.2.6 in (/var/www/html/vendor/htmlawed/htmlawed)
phpmailer/phpmailer version 6.6.0 in (/var/www/html/vendor/phpmailer/phpmailer/src)
simplepie/simplepie version 1.5.8 in (/var/www/html/vendor/simplepie/simplepie/library)
tecnickcom/tcpdf version 6.4.4 in (/var/www/html/vendor/tecnickcom/tcpdf)
michelf/php-markdown in (/var/www/html/vendor/michelf/php-markdown/Michelf)
true/punycode in (/var/www/html/vendor/true/punycode/src)
iamcal/lib_autolink in (/var/www/html/vendor/iamcal/lib_autolink)
sabre/dav in (/var/www/html/vendor/sabre/dav/lib/DAV)
sabre/http in (/var/www/html/vendor/sabre/http/lib)
sabre/uri in (/var/www/html/vendor/sabre/uri/lib)
sabre/vobject in (/var/www/html/vendor/sabre/vobject/lib)
laminas/laminas-i18n in (/var/www/html/vendor/laminas/laminas-i18n/src)
laminas/laminas-servicemanager in (/var/www/html/vendor/laminas/laminas-servicemanager/src)
monolog/monolog in (/var/www/html/vendor/monolog/monolog/src/Monolog)
sebastian/diff in (/var/www/html/vendor/sebastian/diff/src)
donatj/phpuseragentparser in (/var/www/html/vendor/donatj/phpuseragentparser/src/UserAgent)
elvanto/litemoji in (/var/www/html/vendor/elvanto/litemoji/src)
symfony/console in (/var/www/html/vendor/symfony/console)
scssphp/scssphp in (/var/www/html/vendor/scssphp/scssphp/src)
laminas/laminas-mail in (/var/www/html/vendor/laminas/laminas-mail/src/Protocol)
laminas/laminas-mime in (/var/www/html/vendor/laminas/laminas-mime/src)
rlanvin/php-rrule in (/var/www/html/vendor/rlanvin/php-rrule/src)
blueimp/jquery-file-upload in (/var/www/html/vendor/blueimp/jquery-file-upload/server/php)
ramsey/uuid in (/var/www/html/vendor/ramsey/uuid/src)
psr/log in (/var/www/html/vendor/psr/log/Psr/Log)
psr/simple-cache in (/var/www/html/vendor/psr/simple-cache/src)
psr/cache in (/var/www/html/vendor/psr/cache/src)
league/csv in (/var/www/html/vendor/league/csv/src)
mexitek/phpcolors in (/var/www/html/vendor/mexitek/phpcolors/src/Mexitek/PHPColors)
guzzlehttp/guzzle in (/var/www/html/vendor/guzzlehttp/guzzle/src)
guzzlehttp/psr7 in (/var/www/html/vendor/guzzlehttp/psr7/src)
glpi-project/inventory_format in (/var/www/html/vendor/glpi-project/inventory_format/lib/php)
wapmorgan/unified-archive in (/var/www/html/vendor/wapmorgan/unified-archive/src)
paragonie/sodium_compat in (/var/www/html/vendor/paragonie/sodium_compat/src)
symfony/cache in (/var/www/html/vendor/symfony/cache)
html2text/html2text in (/var/www/html/vendor/html2text/html2text/src)
symfony/css-selector in (/var/www/html/vendor/symfony/css-selector)
symfony/dom-crawler in (/var/www/html/vendor/symfony/dom-crawler)
twig/twig in (/var/www/html/vendor/twig/twig/src)
twig/string-extra in (/var/www/html/vendor/twig/string-extra)
symfony/polyfill-ctype not found
symfony/polyfill-iconv not found
symfony/polyfill-mbstring not found
symfony/polyfill-php80 not found
symfony/polyfill-php81 not found
```

</details>

<details>
<summary>SQL replicas</summary>

```
Not active
```

</details>

<details>
<summary>Notifications</summary>

```
Way of sending emails: PHP
```

</details>

<details>
<summary>Plugins list</summary>

```
```

</details>

**Comment by cconard96** [comment]
Please provide the entries from the GLPI logs. The PHP errors get logged to them, and not the Apache error log.
Specifically, please look for entries around the time you tried viewing the dashboards in `files/_log/php-errors.log` and `files/_log/sql-errors.log`.

**Comment by cconard96** [comment]
Also, when there is an error with a stack trace in the browser console like this, being in debug mode may get a cleaner error trace for us.

**Comment by AKorezin** [comment]
Here you are!
![Browser screenshot with enabled debug](https://user-images.githubusercontent.com/5399964/173195438-205d2d33-2062-42c2-9c46-007cf7afd1da.png)
[Log from browser console with debug as in screenshot](https://github.com/glpi-project/glpi/files/8884288/browser.log)
[Log from files/_log/cron.log](https://github.com/glpi-project/glpi/files/8884289/cron.log)
[Log from files/_log/event.log](https://github.com/glpi-project/glpi/files/8884290/event.log)
[Log from files/_log/php-errors.log](https://github.com/glpi-project/glpi/files/8884291/php-errors.log)
There are no more logs there.

I think there's no need to attach Apache access log again.

**Comment by miguelanruiz** [comment]
Hey,

I got same errors, but logs looks not have more information

I can find this console output, can it be some help ?

![image](https://user-images.githubusercontent.com/49277353/174387654-a451f425-bd94-471b-89a4-2fae8e95f585.png)

Steps to reproduce:

1. At first load got:
![image](https://user-images.githubusercontent.com/49277353/174387776-356369c6-b584-4f51-8cd9-f7855a05c4eb.png)

2. Resizing: 
![image](https://user-images.githubusercontent.com/49277353/174387850-1ffbf7c4-7265-4771-9149-10b87341532a.png)

3. After resize (fixed):
![image](https://user-images.githubusercontent.com/49277353/174387921-2a75ea81-f1db-45e0-b7cd-96655274387b.png)

**Comment by miguelanruiz** [comment]
DOMEval definition:
![image](https://user-images.githubusercontent.com/49277353/174390184-b0bf5382-6289-4f52-a5c5-2450e09855aa.png)

done function line:
![image](https://user-images.githubusercontent.com/49277353/174390324-cdf8b712-11ef-4056-8569-1ad253c1e947.png)

**Comment by miguelanruiz** [comment]
Using GLPI 10.0.1, upgrading 

Before apply ([#11851](https://github.com/glpi-project/glpi/pull/11851)) console error was:
![image](https://user-images.githubusercontent.com/49277353/174405933-ee098329-e2c0-4bb7-9790-c18b478ca49c.png)

But after:
There are some extrange lines:
![image](https://user-images.githubusercontent.com/49277353/174406049-01c0dbe1-f3b1-4172-a98e-8508aedbdddc.png)
But error dissapear:
![image](https://user-images.githubusercontent.com/49277353/174406254-8996d0f1-a842-435f-b4dc-260977c498dd.png)

**Comment by heikki27** [comment]
I have the same problem with GLPI 10.0.1 

![Capture](https://user-images.githubusercontent.com/83362467/175499004-3338d798-7f64-46f3-ab94-37926a1b612f.PNG)

With these errors :

VM1118:16 Uncaught TypeError: Cannot read properties of undefined (reading 'layout')
    at HTMLDivElement.<anonymous> (<anonymous>:16:29)
    at HTMLDivElement.<anonymous> (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:86049)
    at Function.each (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14:3076)
    at obj.<computed>.each (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14:1448)
    at Object.<anonymous> (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:86030)
    at l (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:25453)
    at Object.fireWith [as resolveWith] (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:26272)
    at C (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:80410)
    at XMLHttpRequest.<anonymous> (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:82986)
(anonymous) @ VM1118:16
(anonymous) @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
each @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14
each @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14
(anonymous) @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
l @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
fireWith @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
C @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
(anonymous) @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
6VM1118:13 Uncaught TypeError: Cannot read properties of undefined (reading 'remove')
    at HTMLDivElement.<anonymous> (<anonymous>:13:33)
    at HTMLDivElement.<anonymous> (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:86049)
    at Function.each (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14:3076)
    at obj.<computed>.each (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14:1448)
    at Object.<anonymous> (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:86030)
    at l (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:25453)
    at Object.fireWith [as resolveWith] (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:26272)
    at C (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:80410)
    at XMLHttpRequest.<anonymous> (base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25:82986)
(anonymous) @ VM1118:13
(anonymous) @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
each @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14
each @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:14
(anonymous) @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
l @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
fireWith @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
C @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
(anonymous) @ base.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:25
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14956)
    at r [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:37088)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
t.quantity @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
t.createChartRect @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
r @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
n @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
e @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14909)
    at r.a [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:46627)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
t.quantity @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
t.createChartRect @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
a @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
n @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
e @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14956)
    at r [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:42241)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
t.quantity @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
t.createChartRect @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
r @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
n @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
e @ chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14956)
    at r [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:37088)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14909)
    at r.a [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:46627)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14956)
    at r [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:42241)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14956)
    at r [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:37088)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14909)
    at r.a [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:46627)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)
chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1 Uncaught TypeError: Cannot read properties of null (reading '1')
    at t.quantity (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:8589)
    at t.createChartRect (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:14956)
    at r [as createChart] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:42241)
    at r.n [as update] (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:22700)
    at r.e (chartist.min.js?v=7e2a8a1437ad2dbead10ebaaa7589bdb1969da70:1:24054)

**Comment by dxdemetriou** [comment]
After refreshing with F5 everything seems ok while you are in the central view, and everything reloads correctly when the "All" is selected and moving back and forth anywhere. If you go to another option (Dashboard, Personal View,..) and go to another place, if you go again to the Home - All, you'll have to press F5 again.

**Comment by AKorezin** [comment]
*10.0.2* Bump

**Comment by stonebuzz** [comment]
Hi @AKorezin, @dxdemetriou @heikki27 @miguelanruiz 

can you try this commit -> https://github.com/glpi-project/glpi/pull/12476/commits/31179f2af03f4770c7076b9fbeaed12e8cacce92

Best regards

**Comment by dxdemetriou** [comment]
> Hi @AKorezin, @dxdemetriou @heikki27 @miguelanruiz
> 
> can you try this commit -> [31179f2](https://github.com/glpi-project/glpi/commit/31179f2af03f4770c7076b9fbeaed12e8cacce92)
> 
> It comes from this PR -> #12476
> 
> Best regards

I had to use this to work (GLPI 10.0.2):
`$more_types = Plugin::doHookFunction("dashboard_types");`

On first load, the "All" tab shows this:
![Screenshot_20220825_102544](https://user-images.githubusercontent.com/41939421/186602828-22648429-5835-457b-b9be-603587c73e58.png)

and needs refresh for showing the whole content (fixed except the Personal View" part that's still messed-up). If not-refreshed the "All" tab, then the "Dashboard" tab is empty (this needed on the first time only, and every time you leave from the dashboard).

While you are on the dashboard and you have refreshed the "All" tab, every time you go from "Dashboard" tab to the "All" tab you have this:
![Screenshot_20220825_102654](https://user-images.githubusercontent.com/41939421/186603423-f284326a-eec4-4b51-b887-76063759119a.png)

which needed to be refreshed, but the "Dashboard" tab is not being be empty (the refreshing needed only for showing the content on the "All" tab)

I'll try more tests later..

**Comment by stonebuzz** [comment]
"Dashboard" tab is well displayed ?

**Comment by dxdemetriou** [comment]
> "Dashboard" tab is well displayed ?

After loading the "All" tab, the "Dashboard" tab becomes empty which is fixed after refreshing. It's like the "All" tab is stealing the content from the "Dashboard" tab.

**Comment by heikki27** [comment]
10.0.2 - Upgrade solve the problem Tx

**Comment by dxdemetriou** [comment]
I'm in GLPI 10.0.2 from their release, the problem still exists. Probably if you have too many tickets and a long history?

**Comment by AKorezin** [comment]
The issue still persists in *10.0.3*.

**Comment by dxdemetriou** [comment]
It's fixed only the network part on the first loading. Everything else is still messed-up until refresh, and the refresh still needed if you go between "All" and "Dashboard", as it seems that each tab takes the content of the other tab each time.

**Comment by miguelanruiz** [comment]
I agree with @dxdemetriou , it looks like loading to the new dashboard, it tries to take data from the old one. Everything is still messy until it is refresh.

**Comment by AKorezin** [comment]
The issue persists in 10.0.5.

**Comment by cedric-anne** [comment]
Issue is probably fixed by #12953. I close the isue, feel free to reopen if problem persist on [nightly build](https://nightly.glpi-project.org/glpi/).

**Comment by dxdemetriou** [comment]
Tried only the patches on current stable GLPI 10. The ordering is almost solved but still some big widgets are partially overlapping some others. The returning from "All" tab to "Dashboard" tab still gives empty dashboard and needs refreshing after each returning (probably fixed on latest nightbuild?).

**Comment by dxdemetriou** [comment]
Everything not in dashboard tab are ordered in the correct way now, but everything in the dashboard tab are overlapped on the first line.

**Comment by dxdemetriou** [comment]
https://user-images.githubusercontent.com/41939421/206724097-d38364e6-be82-4c26-ae64-c9ad449bf751.mp4

**Comment by mostuni** [comment]
Hello, I tried the Nightly Builds 10.0-618e2f5 version and the overlapping problem was solved. Today I saw that there is a new one, I'm going to try again. regards

**Comment by AKorezin** [comment]
Fixed in 10.0.6.
